### DISCUSSION TITLE
'Dialog' object has no attribute 'wm_attributes'

### DISCUSSION THREAD

**Discussion by Linos1391** [alejandroautalan/pygubu-designer#273] [Q&A] [answered]
Hey,

First, pygubu is amazing, I have been enjoying it so far.

## Code

```
self.mainwindow = builder.get_object('title', master)
self.advance_dialog = self.builder.get_object('advance', self.mainwindow)
builder.connect_callbacks(self)

self.mainwindow.wm_attributes('-topmost', True)
self.advance_dialog.wm_attributes('-topmost', True)
```

## Return

```
AttributeError: 'Dialog' object has no attribute 'wm_attributes'
```

## Problem

I think you already know the stuff I met. The main window is on top, so is the dialog.

As I knew, this is a widget from pygubu. Hence, I took a look at your 'dialog.py' file but found no function that solve my problem. I don't really want to touch the Toplevel yet.

Thanks.

**Comment by alejandroautalan** [comment, marked as answer]
Hello @Linos1391, thanks for trying pygubu. Sorry for the late response.

The Dialog widget has a _toplevel_ attribute to manage the dialog window.

Use this attribute to modify attributes of the window. In your example:

    self.mainwindow.wm_attributes('-topmost', True)
    self.advance_dialog.toplevel.wm_attributes('-topmost', True)

You have a similar example [here](https://github.com/alejandroautalan/pygubu-designer/blob/01489b9b7ddc18eabce455fe58608ee1b9798a42/examples/dialogs/demo5/mainapp.py#L21-L22).

Let me know if this solves your problem.

Regards
Alejandro A.

**Comment by Linos1391** [comment]
Oh, I did not go through the mainapp.py file yet. Thanks for the response, it does not late or anything. Have a nice day anyway.
